pkg/kn/commands/service/configuration_edit_flags.go

```diff
@@ -165,3 +175,15 @@ func (p *ConfigurationEditFlags) computeResources(resourceFlags ResourceFlags) (
 
 	return resourceList, nil
 }
+
+func (p *ConfigurationEditFlags) mapFromArray(arr []string, delimiter string, flag string) (map[string]string, error) {
+	returnMap := map[string]string{}
+	for _, pairStr := range arr {
+		pairSlice := strings.SplitN(pairStr, delimiter, 2)
+		if len(pairSlice) <= 1 {
+			return nil, fmt.Errorf("%s argument requires a value that contains the '=' character; got %s", flag, pairStr)
```

Comment:
use %q instead so it quotes them, otherwise sometimes that work plus the rest of the sentence could make sense as an english sentence and confuse people.

Reply:
I didn't love having to pass the flag but wasn't aware wrapping was an option. Thanks for the suggestion!